pkg/image/api/types.go

```diff
@@ -50,6 +50,12 @@ type Image struct {
 	DockerImageManifest string
 	// DockerImageLayers represents the layers in the image. May not be set if the image does not define that data.
 	DockerImageLayers []ImageLayer
+	// DockerImageSignatures provides the signatures on this JWS as opaque blobs.
+	DockerImageSignatures [][]byte
+	// DockerImageManifestMediaType specifies the mediaType of manifest.
+	DockerImageManifestMediaType string
+	// DockerConfigImage is a JSON blob that the runtime uses to set up the container.
+	DockerConfigImage string
```

Comment:
Can we call this DockerImageConfig then, and give it a description
that indicates that (specifically, that it's the value extracted from
the manifest)?